plugins/modules/sp_client_policy.py

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+# (c) 2024,Tom page <tpage@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1', 'status': ['preview'], 'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: sp_client_policy
```

Comment:
I think the module should really be called `schedule` as that is the object you are setting the state of